pydra/engine/specs.py

```diff
@@ -636,11 +636,12 @@ def _check_requires(self, fld, inputs):
 
         if "requires" in fld.metadata:
             # if requires is a list of list it is treated as el[0] OR el[1] OR...
-            if all([isinstance(el, list) for el in fld.metadata["requires"]]):
-                field_required_OR = fld.metadata["requires"]
+            required_fields = list(fld.metadata["requires"])
```

Comment:
What if we get a string? This will turn `"field"` into `['f', 'i', 'e', 'l', 'd']`.

```suggestion
            required_fields = ensure_list(fld.metadata["requires"])
```

Or, if we don't want to allow this:

```suggestion
            if isinstance(fld.metadata["requires"], str):
                raise ValueError(...)
            required_fields = list(fld.metadata["requires"])
```

Reply:
Since `"requires": "foo"` makes sense, maybe we could allow it? Then, `ensure_list` would be the solution.